compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -608,6 +608,25 @@ pub fn linking_symbol_name_for_instance_in_crate<'tcx>(
     format!("{prefix}{undecorated}{suffix}{args_in_bytes}")
 }
 
+pub fn exporting_symbol_name_for_instance_in_crate<'tcx>(
```

Comment:
I should do same with `linking_symbol_name_for_instance_in_crate`, otherwise symbol cannot be imported from dynamic library.